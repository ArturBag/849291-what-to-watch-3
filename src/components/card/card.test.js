import React from "react";
import renderer from "react-test-renderer";
import Card from "./card.jsx";


it(`Render Card`, () => {

  const tree = renderer
    .create(<Card
      title={`Home Alone`}
      preview={`img/test.jpg`}
      videoSrc={`https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`}
      id={0}
      onMovieCardTitleClick={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();

});
