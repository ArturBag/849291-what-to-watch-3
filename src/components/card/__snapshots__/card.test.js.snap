// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Card 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseOut={[Function]}
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Home Alone"
      height={175}
      src="img/test.jpg"
      width={280}
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
      onClick={[Function]}
    >
      Home Alone
    </a>
  </h3>
</article>
`;
